example/app/page.tsx

```diff
@@ -29,8 +29,8 @@ const Home = () => {
           <List>
             {EXAMPLES.map((id) => (
               <ListItem key={id}>
-                <ListItemButton>
-                  <Link href={`/${id}`}>{id}</Link>
+                <ListItemButton LinkComponent={Link} href={`/${id}`}>
```

Comment:
What is that?

Reply:
It makes link clickable on full button width, not only text